dev/AppNotifications/AppNotificationActivatedEventArgs.cpp

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "AppNotificationActivatedEventArgs.h"
+#include "Microsoft.Windows.AppNotifications.AppNotificationActivatedEventArgs.g.cpp"
+#include "AppNotificationBuilder/AppNotificationBuilderUtility.h"
+
+namespace winrt::Microsoft::Windows::AppNotifications::implementation
+{
+    winrt::Windows::Foundation::Collections::IMap<hstring, hstring> AppNotificationActivatedEventArgs::DecodeArguments(std::wstring arguments)
```

Comment:
this needs a comment. What is the expected input and output?

A sample would be great

Reply:
The samples will be updated to use the builder next.

Reply:
no i meant a comment

Reply:
This API has a contract. It is important to comment that contract here so that others don't come in and introduce bugs.

Reply:
ok, I'll add a comment in the nits PR today. I don't want to push new changes to this one, unless something is really brocken.